### ISSUE TITLE
Sidebar IDs

### ISSUE THREAD

**Issue by justintadlock** [samikeijonen/checathlon#11]
I know it's a recommendation on WordPress.com, but I dislike naming sidebar IDs like `sidebar-1`, `sidebar-2`, and so on.  It's OK with 3 or 4 sidebars, but when you've got 10 sidebars, it gets a bit confusing to follow in the code.  Named sidebars like `downloads` or `sidebar-downloads` is far easier to catch onto quickly.

Anyway, this is just purely a different developer perspective that I thought I'd throw out there.

**Comment by samikeijonen** [comment]
Yeah it was originally for wp.com but theme was not approved in there. So
might name them more sane way.

On Dec 2, 2016 23:11, "Justin Tadlock" <notifications@github.com> wrote:

I know it's a recommendation on WordPress.com, but I dislike naming sidebar
IDs like sidebar-1, sidebar-2, and so on. It's OK with 3 or 4 sidebars, but
when you've got 10 sidebars, it gets a bit confusing to follow in the code.
Named sidebars like downloads or sidebar-downloads is far easier to catch
onto quickly.

Anyway, this is just purely a different developer perspective that I
thought I'd throw out there.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/samikeijonen/checathlon/issues/11>, or mute the thread

**Comment by justintadlock** [comment]
Not approved on .COM?  That's crazy to me. :)

**Comment by samikeijonen** [comment]
Yeah same old story. Not good enough design and our users brobably don't
like it.

On Dec 2, 2016 11:17 PM, "Justin Tadlock" <notifications@github.com> wrote:

Not approved on .COM? That's crazy to me. :)

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/samikeijonen/checathlon/issues/11#issuecomment-264564944>,
or mute the thread
